app/src/org/commcare/engine/resource/installers/SingleAppInstallation.java

```diff
@@ -74,7 +75,9 @@ protected void deliverResult(CommCareSetupActivity receiver,
                     @Override
                     protected void deliverUpdate(CommCareSetupActivity receiver,
                                                  int[]... update) {
-                        receiver.updateResourceProgress(update[0][0], update[0][1], update[0][2]);
+                        if (!CommCareApplication._().isConsumerApp()) {
```

Comment:
since we're trying to obfuscate away as much of the underlying process as possible for consumer apps, we don't want to change what the dialog says based on these updates. This will instead just leave up the generic "Initializing your application" message.